airtable/plugin.py

```diff
@@ -37,7 +37,10 @@ def airtable_url_to_ids(url: str) -> Tuple[AirtableBaseID, AirtableTableID]:
         raise ValueError("Invalid Airtable URL")
 
     base_id = match.group(1)
-    table_id = match.group(2) if match.group(2) else None
+    # When loading the Airtable using the URL that only has the base id (e.g. https://airtable.com/apptpbyiHjjfPecFw)
+    # Airtable automatically drops you into the first table of the base and we don't expect the table_id to be an empty string
+    # if the user copies the URL from the browser.
+    table_id = match.group(2) if match.group(2) else ""
```

Comment:
Should this not be an error?  I am confused at what is being fixed here.  If we expect this to be useful for URLs that only have a base ID, should we not handle that gracefully?

Reply:
The AirtableTableID has `id` of type `str`: https://github.com/d8e-ai/lutra-plugin/blob/38b5923ffcc4d1d555ac0ce51f589adbf57edda0/airtable/plugin.py#L20

Prior to this change, there was a type issue where we set the `id` to `None` when `id` should have been a string that is not optional.

Reply:
If we think the URLs should **always** contain the table id, I'm with Jerry and think this should raise an exception.

Reply:
i.e., we only support URLs with both base and table ids for this function